other/perun-propagate/etc/perun_propagate

```diff
@@ -29,6 +29,16 @@ fi
 # Load lsb functions
 . /lib/lsb/init-functions
 
+# Return 1 if process with PID is still runing, 0 if not
```

Comment:
Usually return 0 means everything is OK, return non-zero means that something is wrong or at least it isn't as you expect (e.g. process is not running).